src/pages/Quiz/DisplayQuestion.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { ReactElement } from 'react';
+import { Text, Flex, Button } from '@chakra-ui/react';
+import { useBookmark, useQuiz } from '../../contexts';
+import { VALIDATE_ANSWER } from '../../contexts/quiz-context/quiz-reducer';
+
+export function DisplayQuestion(): ReactElement {
+  const { state, dispatch } = useQuiz();
+  const { isAnswerSelected, selectedAnswer, words, currentWordIndex } = state;
+  const { bookmarkedWords, bookmarkWord, removeBookmark } = useBookmark();
+  function setColor(index: number): string {
+    if (isAnswerSelected) {
+      switch (index) {
+        case words[currentWordIndex].mcq.answer:
+          return 'green';
+        case selectedAnswer:
+          return 'red';
+        default:
+          return 'gray';
+      }
+    }
+    return 'gray';
+  }
+  function setVariant(index: number): string {
+    if (isAnswerSelected) {
+      return index === selectedAnswer || index === words[currentWordIndex].mcq.answer
```

Comment:
break down the conditional statement of the ternary operator into simpler vars and give meaningful names. For making if statement more understandable.
```suggestion
      const isUserSelectedAnswer = index === selectedAnswer;
      const isCurrentAnswerCorrect = index === words[currentWordIndex].mcq.answer
      
      return isUserSelectedAnswer ||  isCurrentAnswerCorrect
```